functions/src/services/user/defaultUserService.ts

```
@@ -0,0 +1,137 @@
+    const subcollections = [
+      "stepCount",
+      "bodyWeight",
+      "heartRate",
+      "questionnaireResponses",
+      "messages",
+    ];
+
+    for (const collection of subcollections) {
+      const collectionRef = userRef.collection(collection);
+      const docs = await collectionRef.get();
+      docs.docs.forEach((doc) => batch.delete(doc.ref));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)